pkg/cmd/version/version.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2024 ScyllaDB
+
+package version
+
+import (
+	"fmt"
+
+	"github.com/scylladb/scylla-operator/pkg/genericclioptions"
+	"github.com/scylladb/scylla-operator/pkg/version"
+	"github.com/spf13/cobra"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+type Options struct {
+}
+
+func NewOptions(streams genericclioptions.IOStreams) *Options {
+	return &Options{}
+}
+
+func NewVersionCmd(streams genericclioptions.IOStreams) *cobra.Command {
```

Comment:
nit: NewCommand
since it has its own package

Reply:
done